specification/communication/data-plane/NetworkTraversal/stable/2022-03-01/CommunicationNetworkTraversal.json

```
@@ -0,0 +1,179 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CommunicationNetworkTraversalClient",
+    "description": "Azure Communication Network Traversal Service",
+    "version": "2022-03-01"
+  },
+  "paths": {
+    "/networkTraversal/:issueRelayConfiguration": {
+      "post": {
+        "tags": [
+          "Turn"
+        ],
+        "summary": "Issue a configuration for an STUN/TURN server.",
+        "operationId": "CommunicationNetworkTraversal_IssueRelayConfiguration",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "description": "Optional request for providing the id and/or route type for the returned relay configuration.",
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/CommunicationRelayConfigurationRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/CommunicationRelayConfiguration"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Issue Relay Configuration": {
+            "$ref": "./examples/IssueRelayConfiguration.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CommunicationRelayConfigurationRequest": {
+      "description": "Request for a CommunicationRelayConfiguration.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "An identity to be associated with telemetry for data relayed using the returned credentials. Must be an existing ACS user identity. If not provided, the telemetry will not contain an associated identity value.",
+          "type": "string"
+        },
+        "routeType": {
+          "description": "Filter the routing methodology returned. If not provided, will return all route types in separate ICE servers.",
+          "$ref": "#/definitions/RouteType"
+        },
+        "ttl": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The credential Time-To-Live (TTL), in seconds. The default value (172800) will be used if given value exceeds it."
```

You can specify the default value in OpenAPI with the "default" property:
```suggestion
          "description": "The credential Time-To-Live (TTL), in seconds. The default value (172800) will be used if given value exceeds it.",
          "default": 172800
```